packages/calcite-components/src/components/flow/flow.tsx

```diff
@@ -61,6 +61,19 @@ export class Flow implements LoadableComponent {
     return activeItem?.setFocus();
   }
 
+  // --------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * This property enables the component to consider other elements implementing flow-item's interface.
+   *
+   * @internal
+   */
+  @Prop() extraItemSelector: string;
```

Comment:
I think we'll need to decide on this name so it can be standardized ™️ and used for any components that may need to adopt this pattern.

Would we want to expose the fact that this is a selector or just allow other item types?

Maybe this should be an array? Then we can just join them with a comma? Like `extraItems: string[]`

Reply:
Agreed on standardizing. I plan to update the conventions doc with info on this.

I went with the current name because it captures what it'll be used for and the type of value that's expected. 

WRT the type, I think we should leave it as a string to keep it simple on both sides and because it would also align with DOM-querying APIs where a string `selectors` argument supports one or more selectors. 

Based on the above and your suggestions, I kind of like `customItemSelector(s)` since it conveys the use case better than the current name.

Alternatively, we can buy some extra time to think about alternative names if we keep everything internal for this release.

Reply:
Sounds good!